### DISCUSSION TITLE
💨给你的Nanopi R4S手动超频

### DISCUSSION THREAD

**Discussion by giscus-kuokuo-io** [nagaame/giscus-discussions#13] [Announcements]
# 💨给你的Nanopi R4S手动超频

前言
我的nanopi R4S安装的openwrt系统默认给R4S的频率比较保守, 刷超频固件或者自己修改固件都比较麻烦, 我们可以手动超频率,
超频后可以让它跑到稳定的千兆速度, 而且我的散热盒子自带了一个小风扇, 散热应该是有保障的, 不用担心过热

> ⚠️超频有风险, 修改错误可能会导致无法启动系统, 并且超频后, 需要的电压较高, 你需要注意你的电源电压是否足够, 建议电源输出功率15-20W,
电压不够可能会导致系统不稳定, 自动关机等, 需要自己准备好点的电源

修改
首先挂载一下必要的文件

我们先挂载/dev/mmcblk0p1

$ mount /dev/mmcblk0p1 /mnt/mmc1

复制dtb文件到home目录

$ cp /mnt/mmc1/rockchip.dtb ~

回到home目录, 反编译dtb为dts

$ opkg update
# 安装dtc
$ opkg install dtc
# 反编译dtb
$ dtc -I dtb -O dts -o rockchip.

https://kuokuo.io/2022/03/28/boost-nanopi-r4s/

**Comment by daiaji** [comment]
风冷能压吗？

**Reply by nagaame** [reply]
风冷可以的, 如果是被动散热, 夏天温度会飙到60度到70度, 不是很推荐, 建议加个风扇
